### ISSUE TITLE
pre-training

### ISSUE THREAD

**Issue by xiaowowwwww** [Dc-huang/Dual-channel-NIR-denosing#2]
Hello, I am very confused about the training, is there no pre-training part here? If you can, can you give some pre-training source code, really look forward to your reply, thank you!

**Comment by Dc-huang** [comment]
Pre-training requires you to modify the loss function. The specific method is to use l2_loss to train the network. The parameters are the intput of the net and the sum of the output of the two branches.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Dc-huang/Dual-channel-NIR-denosing"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月6日(星期四) 下午2:35
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[Dc-huang/Dual-channel-NIR-denosing] pre-training (Issue #2)





 
Hello, I am very confused about the training, is there no pre-training part here? If you can, can you give some pre-training source code, really look forward to your reply, thank you!

**Comment by xiaowowwwww** [comment]
Thank you！
